src/pd/pd.rs

```diff
@@ -544,7 +544,12 @@ impl<T: PdClient> Runnable<Task> for Runner<T> {
             } => {
                 let approximate_size = match region_size {
                     Some(size) => size,
-                    None => get_region_approximate_size(&self.db, &region).unwrap_or(0),
+                    None => {
```

Comment:
It's weird that you change this but don't use the result.
I think you can just report it to PD in this PR.